nornir/core/processor.py

```diff
@@ -1,5 +1,7 @@
 from typing import List, Protocol
 
+from typing_extensions import Protocol
```

Comment:
@nemith It looks like you need to rebase to get the latest changes that I released last night.